binary_test.go

```diff
@@ -124,3 +124,27 @@ func TestBinary(t *testing.T) {
 	testDecodeFramework(t, "replyBinary_65536", []byte(s65560[:65536]))
 	testDecodeFramework(t, "replyBinary_null", nil)
 }
+
+func TestBinaryEncode(t *testing.T) {
+	s0 := ""
+	s1 := "0"
+	s16 := "0123456789012345"
+
+	s1024 := ""
+	for i := 0; i < 16; i++ {
+		s1024 += fmt.Sprintf("%02d 456789012345678901234567890123456789012345678901234567890123\n", i)
+	}
+
+	s65560 := ""
+	for i := 0; i < 1024; i++ {
+		s65560 += fmt.Sprintf("%03d 56789012345678901234567890123456789012345678901234567890123\n", i)
+	}
+
+	testJavaDecode(t, "argBinary_0", []byte(s0))
```

Comment:
`argBinary_0` response binary array with one zero byte:
```
  public Object replyBinary_0()
  {
    return new byte[0];
  }
```

Reply:
En, I may not understand what's your meaning. Because `s0 := ""`,  I think `[]byte(s0)` in go equals `new byte[0]` in java.